plugins/SliceInfoPlugin/SliceInfo.py

```diff
@@ -108,4 +108,4 @@ def _onWriteStarted(self, output_device):
             Logger.log("i", "Sent anonymous slice info to %s", self.info_url)
             f.close()
         except Exception as e:
-            Logger.logException("e", e)
+            Logger.log("e", "Exception occured: %s" %(e))
```

Comment:
The output in the console will say it comes from SliceInfo.py, so I think it is enough.